test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'cli.js'
+);
+
+const args = [
+  npmPath,
+  'install'
+];
+
+let stdout = '';
+let stderr = '';
+
+const proc = spawn(process.execPath, args, {
+  stdio: 'pipe',
+  cwd: common.tmpDir
+});
+
+proc.stdout.setEncoding('utf8');
+proc.stderr.setEncoding('utf8');
+
+proc.stdout.on('data', (data) => stdout += data);
+proc.stdout.on('err', (data) => stderr += data);
```

Comment:
Why bothering to capture `stdout` and `stderr` if nothing will be done with them?